CONTRIBUTING.rst

```diff
@@ -43,7 +43,7 @@ Development
 Requirements
 -------------
 
-- Python >=3.9
+- Python >=3.10
```

Comment:
I'm in favor of not using `match` now, and we can support some Python versions such as 3.8 (2024), 3.9 (2025). https://devguide.python.org/versions/
WDYT @KAUTH

Reply:
I believe since we are not using any _key_ features of the later Python versions and we are not impacted by missing out on their improved performance either, we can support from version 3.8.
My assumption was that we didn't want to in the past based on the `Pipfile` and tox `envlist`. So I would suggest handling in this PR all the discrepancies found about the supported Python versions in this package:
1. `CONTRIBUTING.rst` - `Python >=3.8`
2. `pyproject.toml` - `requires-python = ">=3.8"`
3. `Pipfile` - `python_version = "3.8"; `pipenv` [does not support multiple Python versions](https://github.com/pypa/pipenv/issues/1050)
4. In `tox.ini` we should add to the `envlist` `py38` and `py39`, we want to test them if we are going to support them

I see at least these for the moment

Reply:
Hi @deepNet-Chirag,
I think @KAUTH's comments are an excellent way to go. Feel free to let me know if you need some help.

Reply:
You can try to run `make reformat`. It will automatically fix most of the issues.
Also you can check the linting locally using `tox -e lint`